package.json

```diff
@@ -27,10 +27,11 @@
   "author": "Chris Stoll",
   "license": "MIT",
   "dependencies": {
-    "bunsen-core": "0.12.1",
+    "bunsen-core": "^0.13.1",
```

Comment:
I don't believe the `^` does anything for you here since it is a pre 1.0 release.